File: module_system/src/main/java/com/lcworld/module_system/activity/ToolEntranceActiviy.java
package com.lcworld.module_system.activity;

import android.os.Bundle;
import com.lcworld.library_base.base.BaseActivityEnhance;
import com.lcworld.library_base.base.BaseViewModelEnhance;
import com.lcworld.module_system.R;
import com.lcworld.module_system.databinding.SystemActivityEntranceBinding;
import me.goldze.mvvmhabit.BR;

/**
 * 工具
 */
public class ToolEntranceActiviy extends BaseActivityEnhance<SystemActivityEntranceBinding, BaseViewModelEnhance> {
    @Override
    public int initContentView(Bundle bundle) {
        return R.layout.system_activity_entrance;
    }

    @Override
    public int initVariableId() {
        return BR.viewModel;
    }
}

File: module_system/src/main/java/com/lcworld/module_system/activity/ToolEntranceActiviy.kt
package com.lcworld.module_system.activity

import android.os.Bundle
import com.lcworld.library_base.base.BaseActivityEnhance
import com.lcworld.library_base.base.BaseViewModelEnhance
import com.lcworld.module_system.R
import com.lcworld.module_system.databinding.SystemActivityEntranceBinding
import me.goldze.mvvmhabit.BR

/**
 * 工具
 */
class ToolEntranceActiviy : BaseActivityEnhance<SystemActivityEntranceBinding, BaseViewModelEnhance>() {
    override fun initContentView(bundle: Bundle): Int {
        return R.layout.system_activity_entrance
    }

    override fun initVariableId(): Int {
        return BR.viewModel
    }
}
